src/DataBaseInfo.hs

```diff
@@ -0,0 +1,24 @@
+module DataBaseInfo where
+
+import Database.PostgreSQL.Simple
+import LoadEnv
+import System.Environment
+import Data.Word
+
+connInfo :: IO ConnectInfo
+connInfo = do
+  loadEnv
```

Comment:
@kinorius123 I think the function that load the envs in the `.env` file should be executed in a different file.

Reply:
@kinorius123 To be precies, I think you should load the envs in the `Main.hs` module.